engine/g5k_engine.py

```diff
@@ -109,6 +109,10 @@ def get_job(self):
             self._make_reservation()
         else:
             logger.info("Using running oargrid job %s" % style.emph(self.gridjob))
+            
+
+        # Wait for the job to start
+        EX5.wait_oargrid_job_start(self.gridjob)
```

Comment:
I like the idea,  but I'm not sure that the behaviour of `wait_oargrid_job_start` is exactly what we expect here.  We should make sure that when the function returns the job is really in `running` state. 
In some cases, nodes may be late to be acquired for the current job (e.g reboot after the previous job). So I've the feeling that some precaution should be taken to really ensure that the job is in running state.
One possibility would be to loop over several `wait_oargrid_job_start` and testing the job state every time.

Of course if the function behaves as its name pretend there is no issue here.

Additionnally I think we don't need to loop anymore in the followig lines to get the nodes of the job.